src/main/java/net/rptools/maptool/client/functions/MapFunctions.java

```diff
@@ -280,21 +282,22 @@ public Object childEvaluate(
 
       if (config.has("map asset")) {
         final var mapAssetId = config.getAsJsonPrimitive("map asset").getAsString();
-        final var mapAssetKey = FunctionUtil.getAssetKeyFromString(mapAssetId);
```

Comment:
If we instead change `FunctionUtil.getAssetKeyFromString()`, than all callers will support `lib:` URIs. I think that's just here and some table functions, but the point stands :slightly_smiling_face:

Reply:
That is smarter, and now done. Rewrote my previous PR for Input functions to use the same.